client/src/app/core/repositories/assignments/assignment-repository.service.ts

```diff
@@ -219,8 +219,6 @@ export class AssignmentRepositoryService extends BaseIsAgendaItemAndListOfSpeake
      * @param originalPoll the original poll
      */
     public async updateVotes(poll: Partial<AssignmentPoll>, originalPoll: ViewAssignmentPoll): Promise<void> {
-        poll.options.sort((a, b) => a.weight - b.weight);
```

Comment:
dunno why that was here itfp. Any ideas @FinnStutzenstein ?

Reply:
It was originated by the initial assignment-implementation (#4537). So a never-checked bug since then.